src/Auth0/Login/Auth0JWTUser.php

```diff
@@ -2,99 +2,35 @@
 
 namespace Auth0\Login;
 
+use Auth0\Login\Traits\Auth0UserTrait;
+use Illuminate\Contracts\Auth\Authenticatable;
+
 /**
  * This class represents a generic user initialized with the user information
  * given by Auth0 and provides a way to access to the decoded JWT data.
  */
-class Auth0JWTUser implements \Illuminate\Contracts\Auth\Authenticatable
+class Auth0JWTUser implements Authenticatable
 {
-    private $userInfo;
+
+    use Auth0UserTrait;
 
     /**
      * Auth0JWTUser constructor.
      *
-     * @param $userInfo
+     * @param array $userInfo
      */
-    public function __construct($userInfo)
+    public function __construct(array $userInfo)
```

Comment:
Breaking change. The userinfo that's generated comes from ID token validation/decoding, which, in the PHP SDK, returns an array. We were converting to an object to pass, then an array to use.